ItsME/Presentation/Scenes/TotalCV/CVEditing/CategoryEditing/CategoryEditingViewController.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  CategoryEditingViewController.swift
+//  ItsME
+//
+//  Created by MacBook Air on 2023/03/03.
+//
+
+import UIKit
+import Then
+import SnapKit
+
+class CategoryEditingViewController: UIViewController, UITableViewDataSource, UITextFieldDelegate {
```

Comment:
클래스가 상속될 여지가 없으면 final 을 붙이는게 성능상 이점이 있습니다.
```suggestion
final class CategoryEditingViewController: UIViewController, UITableViewDataSource, UITextFieldDelegate {
```